### PR TITLE
Added aws-request-signing.

### PR DESCRIPTION
**Pull request by DanielRailean**
### Description

What did you change and why?
 
Added a new community plugin to be published on Kong Hub.

### Testing instructions

Netlify link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Comment by acgoldsmith** [comment]
Thanks for your submission, but at the moment, we’re not accepting third-party plugins unless they’re from trusted technical partners. We don't have a timeline right now for when that will change.

**Comment by acgoldsmith** [comment]
Hi @DanielRailean , apologies for miscategorizing this submission.

**Comment by acgoldsmith** [comment]
We're coordinating w/ @andrewgkew for this plugin submission.

**Comment by acgoldsmith** [comment]
Hi @DanielRailean  and @andrewgkew  - you can now see the preview content at https://deploy-preview-5131--kongdocs.netlify.app/hub/thelegogroup/aws-request-signing/ . 

I'll take care of those vale linting items next.

**Comment by DanielRailean** [comment]
Hi @acgoldsmith, let me know if other changes are needed before getting this approved.

File: app/_hub/TheLEGOGroup/aws-request-signing/_index.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+# If your plugin doesn't have an icon yet, you can duplicate the default_icon.png file.
+#
+# 4. Fill in the template in this file.
+#
+# The following YAML data must be filled out as prescribed by the comments
+# on individual parameters. Also see documentation at:
+# https://github.com/Kong/docs.konghq.com/app/_hub for examples.
+# Remove inapplicable entries and comments as needed.
+
+name: aws-request-signing
```
Comment (lena-larionova) [inline app/_hub/TheLEGOGroup/aws-request-signing/_index.md:23]:
```suggestion
name: AWS Request Signing
```

File: app/_hub/TheLEGOGroup/aws-request-signing/_index.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+  # plugin          | extensions of the core platform
+  # integration     | extensions of the Kong Admin API
+
+desc: Sign requests with AWS SIGV4 and temp credentials for secure use of AWS Lambdas in Kong. # (required) 1-liner description; max 80 chars
+description: |
+  This plugin allows for secure communication with AWS Lambdas. It signs requests with AWS SIGV4 and temporary credentials obtained from sts.amazonaws.com using an OAuth token. 
+  This eliminates the need for an AWS API Gateway and simplifies the use of Lambdas as upstreams in Kong. 
+  
+  However, in order to use this plugin, there is an AWS setup required.
+  Specifically, you will need to add your token issuer to the "Identity Providers" in your AWS account, this way the plugin can request temporary credentials. More details can be seen in the [plugin repo.](https://github.com/LEGO/kong-aws-request-signing#aws-setup-required)
```
Comment (acgoldsmith) [inline app/_hub/TheLEGOGroup/aws-request-signing/_index.md:57]:
```suggestion
  Specifically, you will need to add your token issuer to the "Identity Providers" in your AWS account, this way the plugin can request temporary credentials. 
 
 For more information on the required AWS setup, visit the [plugin repo.](https://github.com/LEGO/kong-aws-request-signing#aws-setup-required)
```

**Review by acgoldsmith — APPROVED** [review]
Approving for @andrewgkew to release along with the new plugin

File: app/_hub/TheLEGOGroup/aws-request-signing/_index.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+
+desc: Sign requests with AWS SIGV4 and temp credentials for secure use of AWS Lambdas in Kong. # (required) 1-liner description; max 80 chars
+description: |
+  This plugin allows for secure communication with AWS Lambdas. It signs requests with AWS SIGV4 and temporary credentials obtained from sts.amazonaws.com using an OAuth token. 
+  This eliminates the need for an AWS API Gateway and simplifies the use of Lambdas as upstreams in Kong. 
+  
+  However, in order to use this plugin, there is an AWS setup required.
+  Specifically, you will need to add your token issuer to the "Identity Providers" in your AWS account, this way the plugin can request temporary credentials. 
+ 
+ For more information on the required AWS setup, visit the [plugin repo.](https://github.com/LEGO/kong-aws-request-signing#aws-setup-required)
```
Comment (lena-larionova) [inline app/_hub/TheLEGOGroup/aws-request-signing/_index.md:59]:
```suggestion
  For more information on the required AWS setup, visit the [plugin repo.](https://github.com/LEGO/kong-aws-request-signing#aws-setup-required)
```

**Pull request merged**